js/ui/html_editor/ui.html_editor.js

```diff
@@ -509,22 +509,30 @@ const HtmlEditor = Editor.inherit({
     },
 
     _optionChanged: function(args) {
+        let value;
```

Comment:
Why do we need define it here?

Reply:
because we have [no-case-declarations](https://eslint.org/docs/latest/rules/no-case-declarations) rule. 
Fixed by wrapping into blocks using brackets.